leetcode/src/42.c

```diff
@@ -0,0 +1,34 @@
+int max(int a, int b){

```

Comment:
Could you please add a small description of the algorithm?

Reply:
I added the description